mkdocs.yml

```diff
@@ -299,8 +299,12 @@ extra:
     version:
         provider: mike
     analytics:
-        provider: google
-        property: G-3M31G9B0QJ
+        provider: custom
+        property: !ENV [AMPLITUDE_API_KEY, "8ab9ae57b486baa5d4ab626559fd8106"]
+        url: !ENV [AMPLITUDE_URL, "https://api2.amplitude.com/2/httpapi"]
+        # disabled by default for local development
+        # to enable, set AMPLITUDE_ENABLED=enabled in your environment
+        enabled: !ENV [AMPLITUDE_ENABLED, "disabled"]
```

Comment:
I would suggest enabling this if the API Key is set, so that you don't need a separate enabled/disabled setting. It doesn't make sense to enable Amplitude without an API Key, anyways.

It does mean that for local development we shouldn't have a default API key set in our config file or envrc, but I think that should be acceptable. Most of the time we'll just be testing this in staging or in previews

Reply:
Agreed with this - if we're able to set the API key in the runner we won't need to include it anywhere in source